### DISCUSSION TITLE
Can I use in-line math formula?

### DISCUSSION THREAD

**Discussion by evandde** [hbstack/.github#19] [Q&A]
I want to use in-line math formula like this (https://evanote.mew.kr/post/210924-bithacks-power/).
![image](https://github.com/hbstack/.github/assets/57508537/302b0ab9-2c9a-4f03-91b5-f53f4a6fd355)
(Here, `a`, `n`, or `a^n` is in-line math formula that I want. Sorry that it is a Korean page)

But in HBStack, the katex module looks like to provide only for paragraph type formula.

Is it possible?

**Comment by razonyang** [comment]
Currently, not. But it should be easy to implement, track this on https://github.com/hugomods/katex/issues/9.

**Reply by evandde** [reply]
Currently it seems that double dollar (as below) for paragraph formula is working well, even though there is no instruction in the docs.

```
$$
blah blah
$$
```

It would be helpful to use single dollar (`$ blahblah $`) for inline formula.

**Comment by razonyang** [comment]
> It would be helpful to use single dollar ($ blahblah $) for inline formula.

It's impossible or won't fix, it requires searching the content (which is an expensive job) to indicate whether KaTeX is used, and then include the JS. EDITED: although we could set an extra page parameter like `katex=true` on front matter instead of searching the content, it seems still complex for users (may forget setting that parameter).

The KaTeX module `v0.2.0` supports inline formula, see https://hugomods.com/en/docs/content/katex/#using-katex-via-shortcode.

**Comment by ronibd0** [comment]
> I want to use in-line math formula like this (https://evanote.mew.kr/post/210924-bithacks-power/). ![image](https://user-images.githubusercontent.com/57508537/238582628-302b0ab9-2c9a-4f03-91b5-f53f4a6fd355.png) (Here, `a`, `n`, or `a^n` is in-line math formula that I want. Sorry that it is a Korean page)
> 
> But in HBStack, the katex module looks like to provide only for paragraph-type formula.
> 
> Is it possible?

Hi, it's possible. see this example page I did that successfully  [example page](https://bdmeter.info/blog/2023/06/ac-electricity-the-basic-passive-components-in-ac-circuits/#time-constants) . It's easy. just use the .md file format on the page. 

![image](https://github.com/hbstack/.github/assets/33853447/2bc08fa7-d24a-484a-a626-59e13eb40878)
```
E<sub>C</sub> = E (1 − e<sup>−t/RC</sup> )
```
Solutions: 
```
η = &sum;<sub>i=0</sub> b<sub>i</sub>2<sup>i</sup>, where b<sub>i</sub> = 0 or 1
```

rendered: 

η = &sum;<sub>i=0</sub> b<sub>i</sub>2<sup>i</sup>, where b<sub>i</sub> = 0 or 1
